### PR TITLE
Removing shop=jewellery

### LINKED ISSUE #4962: Remove rendering of shop=jewellery (but keep jewelry)
https://wiki.openstreetmap.org/wiki/Tag:shop%3Djewellery
has been deprecated but the **single existing** node in the osm database is still rendered:

https://www.openstreetmap.org/node/10798568929

image

I have no rendering stack available, so I could only do a blind fix.

Ref deprecation in iD editor since 2019:
https://github.com/openstreetmap/iD/pull/6266

### PR DESCRIPTION
**Pull request by BubbaJuice**
Fixes #4962 

Changes proposed in this pull request:
- Removes specialized rendering of shop=jewellery 

Test rendering at z19 with links to the example places:

[Parramatta, Australia](https://www.openstreetmap.org/#map=19/-33.81790/151.00244)
Before
ZfignBWh

After
b4BdXrtU

### CODE REVIEW

File: project.mml [hunk 1]
```diff
@@ -1681,11 +1681,11 @@ Layer:
...
-                                'garden_centre', 'hairdresser', 'hifi', 'car', 'car_repair', 'bicycle', 'mall', 'pet',
-                                'photo', 'photo_studio', 'photography', 'seafood', 'shoes', 'alcohol', 'gift', 'furniture', 'kiosk',
-                                'mobile_phone', 'motorcycle', 'musical_instrument', 'newsagent', 'optician', 'jewelry', 'jewellery',
+              CASE WHEN shop IN ('supermarket', 'bag', 'bakery', 'beauty', 'bed', 'bookmaker', 'books', 'butcher', 'carpet', 'clothes', 
+                                'computer', 'confectionery', 'fashion', 'convenience', 'department_store', 'doityourself', 'hardware', 
+                                'fabric', 'fishmonger', 'florist', 'garden_centre', 'hairdresser', 'hifi', 'car', 'car_repair', 'bicycle', 
+                                'mall', 'pet', 'photo', 'photo_studio', 'photography', 'seafood', 'shoes', 'alcohol', 'gift', 'furniture', 
+                                'kiosk', 'mobile_phone', 'motorcycle', 'musical_instrument', 'newsagent', 'optician', 'jewelry',
```
Comment (pnorman) [inline project.mml:1688]: Can you remove the rewrapping of the other lines so this part of the diff is just one line?

I'd rather have uneven line lengths than more difficulty with `git bisect`

**Review by pnorman — CHANGES_REQUESTED** [review]
Other than the line reformatting I'm okay with this but have not yet tested.

File: project.mml (hunk 1 above)
Comment (BubbaJuice) [reply project.mml:1688]: Sure, sorry.

**Review by imagico — APPROVED** [review]
Looks good and works as intended.

Before:
before

After:
after

**Review by pnorman — APPROVED** [review]

**Pull request merged**